launcher/ui/pages/instance/ModFolderPage.cpp

```diff
@@ -92,6 +92,10 @@ ModFolderPage::ModFolderPage(BaseInstance* inst, std::shared_ptr<ModFolderModel>
         ui->actionsToolbar->addAction(ui->actionVisitItemPage);
         connect(ui->actionVisitItemPage, &QAction::triggered, this, &ModFolderPage::visitModPages);
 
+        ui->actionRemoveItemMetadata->setToolTip(tr("Remove mod's metadata"));
+        ui->actionsToolbar->insertActionAfter(ui->actionRemoveItem, ui->actionRemoveItemMetadata);
```

Comment:
```suggestion
        ui->actionsToolbar->addAction(ui->actionRemoveItemMetadata);
```
oops, I missed a bit lol

Reply:
I would want it to be near Remove or at least before view folder actions

Reply:
Sorry - oops